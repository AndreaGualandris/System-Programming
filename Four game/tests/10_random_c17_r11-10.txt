3
14
9
5
12
11
9
7
14
14
8
16
10
13
12
4
5
13
9
16
0
15
1
2
0
14
11
7
4
13
13
12
1
4
1
15
14
11
0
13
8
4
9
7
4
16
8
13
4
2
5
13
4
15
14
14
0
15
15
3
13
2
15
16
16
15
9
13
13
11
3
14
2
2
3
4
4
16
3
9
10
12
8
3
2
16
4
14
14
1
3
1
1
3
8
2
9
14
2
8
7
5
13
16
4
15
5
13
13
0
3
10
3
5
7
2
15
13
10
0
4
12
11
4
1
10
1
9
11
6
2
13
15
6
13
15
16
16
15
15
2
11
0
6
4
4
4
10
4
5
0
4
7
2
3
7
9
4
10
15
6
7
8
9
9
6
2
16
5
3
14
8
13
10
14
3
3
1
0
10
6
2
2
4
8
8
1
