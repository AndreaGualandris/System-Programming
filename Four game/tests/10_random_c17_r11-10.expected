-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   o             
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   o          x  
-----------------
01234567890123456
Player 'o' to move:
         o
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   o     o    x  
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   o x   o    x  
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   o x   o  o x  
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   o x   o xo x  
-----------------
01234567890123456
Player 'o' to move:
         o
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
         o       
   o x   o xo x  
-----------------
01234567890123456
Player 'x' to move:
       x
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
         o       
   o x x o xo x  
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
         o    o  
   o x x o xo x  
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o    o  
   o x x o xo x  
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o    o  
   o x xoo xo x  
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o    o  
   o x xoo xo x x
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o    o  
   o x xoooxo x x
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o    o  
   o x xoooxoxx x
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o  o o  
   o x xoooxoxx x
-----------------
01234567890123456
Player 'x' to move:
    x
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
         o  o o  
   oxx xoooxoxx x
-----------------
01234567890123456
Player 'o' to move:
     o
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
     o   o  o o  
   oxx xoooxoxx x
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
              x  
     o   o  oxo  
   oxx xoooxoxx x
-----------------
01234567890123456
Player 'o' to move:
         o
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
         o    x  
     o   o  oxo  
   oxx xoooxoxx x
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
         o    x  
     o   o  oxo x
   oxx xoooxoxx x
-----------------
01234567890123456
Player 'o' to move:
o
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
         o    x  
     o   o  oxo x
o  oxx xoooxoxx x
-----------------
01234567890123456
Player 'x' to move:
               x
               v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
         o    x  
     o   o  oxo x
o  oxx xoooxoxxxx
-----------------
01234567890123456
Player 'x' wins!
